src/main/java/com/questhelper/helpers/quests/atfirstlight/AtFirstLight.java

```diff
@@ -0,0 +1,356 @@
+			"Talk to Guild Scribe Verity behind the bar.");
+		talkToWolf = new NpcStep(this, NpcID.GUILD_HUNTER_WOLF_MASTER, new WorldPoint(1555, 9462, 0),
+			"Talk to Guild Hunter Wolf (Master), next to the bar.");
+		windUpToy = new DetailedQuestStep(this, "Wind up a toy mouse.", toyMouse.highlighted());
+		windUpToy.addIcon(ItemID.TOY_MOUSE_WOUND);
+		useToyOnKiko = new NpcStep(this, NpcID.GUILD_HUNTER_KIKO, new WorldPoint(1552, 9460, 0),
+			"Use the wound up toy mouse on Guild Hunter Kiko near the bar.", toyMouseWound.highlighted());
+		checkBed = new ObjectStep(this, ObjectID.CAT_BED, new WorldPoint(1552, 9460, 0),
+			"Search the cat's bed.");
+		returnToWolf = new NpcStep(this, NpcID.GUILD_HUNTER_WOLF_MASTER, new WorldPoint(1555, 9462, 0),
+			"Return to Guild Hunter Wolf (Master), next to the bar.");
+		goUpTree = new ObjectStep(this, ObjectID.STAIRS_51642, new WorldPoint(1557, 9449, 0),
+			"Go back up the stairs.");
+		talkToFox = new NpcStep(this, NpcID.GUILD_HUNTER_FOX, new WorldPoint(1623, 2982, 0),
+			"Talk to Guild Hunter Fox near the crevice south-east of the Hunter Guild.");
+		takeLeaf = new ObjectStep(this, NullObjectID.NULL_50876, new WorldPoint(1618, 2979, 0),
+			"Search the leafy bush south of the crevice for a smooth leaf.");
+		takeSecondLeaf = new ObjectStep(this, NullObjectID.NULL_50877, new WorldPoint(1673, 2992, 0),
+			"Search the rough-looking bush on the west side of the Locus Oasis.");
+		catchJerboa = new DetailedQuestStep(this, new WorldPoint(1664, 3003, 0),
+			"Catch two Embertailed Jerboa for their tails.", boxTrap.highlighted());
+		catchJerboa.addIcon(ItemID.BOX_TRAP);
+		useTailOnLeaves = new DetailedQuestStep(this, "Use the tails on the leaves.",
+			jerboaTail.highlighted(), smoothLeaf.highlighted());
+		returnToFox = new NpcStep(this, NpcID.GUILD_HUNTER_FOX, new WorldPoint(1623, 2982, 0),
+			"Bring the poultice back to Fox.", makeshiftPoultice);
+		talkToFoxAfterPoultice = new NpcStep(this, NpcID.GUILD_HUNTER_FOX, new WorldPoint(1623, 2982, 0),
+			"Talk to Fox.");
+		talkToAtza = new NpcStep(this, NpcID.ATZA, new WorldPoint(1696, 3063, 0),
+			"Talk to Atza in one of the buildings outside Civitas illa Fortis's south wall, west of the general store.",
+			furSample);
+		talkToAtzaAfterHandingFur = new NpcStep(this, NpcID.ATZA, new WorldPoint(1696, 3063, 0),
+			"Talk to Atza in one of the buildings outside Civitas illa Fortis's south wall, west of the general store.");
+		talkToAtza.addSubSteps(talkToAtzaAfterHandingFur);
+		makeEquipmentPile = new ObjectStep(this, NullObjectID.NULL_52976, new WorldPoint(1697, 3063, 0),
+			"Set-up the pile of equipment next to Atza.", hammer);
+		talkToAtzaForTrim = new NpcStep(this, NpcID.ATZA, new WorldPoint(1696, 3063, 0),
+			"Talk to Atza again for some trimmed fur.");
+		returnToFoxAfterTrim = new NpcStep(this, NpcID.GUILD_HUNTER_FOX, new WorldPoint(1623, 2982, 0),
+			"Return back to Fox to get his report.", trimmedFur);
```

Comment:
full name + direction + removed the "back" word
```suggestion
			"Return to Guild Hunter Fox near the crevice south-east of the Hunter Guild to get his report.", trimmedFur);
```